Queries/NinthQuery.sql

```diff
@@ -0,0 +1,21 @@
+WITH BooksBorrowingFrequency
+AS (
+	SELECT
+	   b.Author, 
+	   count(b.BookID) OVER (PARTITION BY b.Author) AS BorrowingFrequency
+	FROM
+	   Loans AS l
+	JOIN 
+	   Books AS b ON l.BookID = b.BookID
+	)
+
+SELECT
+   bf.Author,
+   bf.BorrowingFrequency,
+   Dense_Rank() OVER (ORDER BY BorrowingFrequency DESC) AS [RankOfAuthor]
+FROM
+   BooksBorrowingFrequency bf
+GROUP BY
```

Comment:
because using window functions doesn't eliminate the redundancy in the (Author ,BorrowingFrequency).
so I wanted just one record from each group of (Author ,BorrowingFrequency).

Before:
![image](https://github.com/user-attachments/assets/be2114bd-71f8-4eef-ab51-2eb6649a34ca)

After:
![image](https://github.com/user-attachments/assets/381dc9f9-c6c4-49c8-93ae-6f2100c2cdbd)